contour/views.py

```diff
@@ -337,13 +337,11 @@ def fit_file(request, pk):
                     latex_string_list = multivariate_distribution.latex_repr(
                         var_symbols
                     )
-                    plotted_figures = PlottedFigure.objects.filter(
-                        probabilistic_model=prob_model
-                    )
+                    srt_figures = sort_plotted_figures(prob_model)
                     return render(request,
                                   'contour/fit_results.html',
                                   {'pk': prob_model.pk,
-                                   'plotted_figures': plotted_figures,
+                                   'srt_figures': srt_figures,
```

Comment:
I did not understand what "srt" stands for. Consider renaming it "sorted_figures".